giskard/llm/__init__.py

```diff
@@ -1,10 +1,11 @@
-from .client import get_default_client, set_default_client, set_llm_api, set_llm_model
+from .client import get_default_client, set_llm_model
+from .embeddings import set_default_embedding, set_embedding_model
 from .errors import LLMImportError
 
 __all__ = [
     "LLMImportError",
     "get_default_client",
-    "set_default_client",
-    "set_llm_api",
```

Comment:
we should not delete `set_default_client` and `set_llm_api`, since it's being referenced on docs: https://docs.giskard.ai/en/stable/open_source/setting_up/index.html